@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,20 @@
+declare interface PMGroupUpdateRequest {
+    prizeCategoryId: string;
+    prizeCatagoryName?: string;
+    prizeCategoryDescription?: string;
+    prizeCategoryOrder?: Array<string>;
```

Comment:
Also, while what you've got for the `prizeCategoryOrder` is 100% accurate, usually convention is to use `[]` instead of `Array<>` because it is shorter, unless you have a long/complex data type for each object and you want to make sure that the scope of the array itself is clear.
```suggestion
    prizeCategoryOrder?: string[];
```